bento/src/main/java/com/yelp/android/bento/core/ViewHolderWrapper.java

```diff
@@ -0,0 +1,44 @@
+package com.yelp.android.bento.core;
+
+
+import android.view.View;
+
+import androidx.recyclerview.widget.RecyclerView;
+
+/**
+ * Wrapper class for ViewHolders that allows {@link ComponentViewHolder}s to have an empty
+ * constructor and perform view inflation post-instantiation. (This is necessary because
+ * RecyclerView.ViewHolder forces an already-inflated view to be passed into the constructor).
+ *
+ * @param <T> The type of data the wrapped {@link ComponentViewHolder} uses.
+ */
+public class ViewHolderWrapper<P, T> extends RecyclerView.ViewHolder {
```

Comment:
Extracted this out from `RecyclerViewComponentController` to make it reusable in `ViewPager2ComponentController`